deploy-as-code/helm/charts/municipal-services/inbox/values.yaml

```diff
@@ -44,6 +44,24 @@ env: |
         key: egov-user
   - name: EGOV_USER_SEARCH_PATH
     value: "user/_search"
+  - name: EGOV_FSM_HOST
+    valueFrom:
+      configMapKeyRef:
+        name: egov-service-host
+        key: fsm
+  - name: EGOV_FSM_FETCHAPPLICATIONIDS_PATH
+    value: "fsm/v1/fetchApplicationIds"
+  - name: EGOV_VEHICLE_HOST
+    valueFrom:
+      configMapKeyRef:
+        name: egov-service-host
+        key: vehicle
+  - name: VEHICLE_SEARCH_PATH
```

Comment:
This is sort of hard coding. If you add value directly like this you cannot overwrite it env to env. And if this value is going to remain constant across env's no need to add it